low-level/framework/utils/utility.py

```diff
@@ -152,14 +152,15 @@ def replace_expr(filename:str, key, new_str:str):
                 f.write(line)
 
     @staticmethod
-    def set_ownership_recursively(root_dir, uid, gid):
+    def set_ownership_recursively(list_root_dir, uid, gid):
```

Comment:
check that no existing functions are affected

Reply:
yes verified